src/utils/pdf/fileUtils.ts

```diff
@@ -0,0 +1,37 @@
+import fs from 'fs';
+import path from 'path';
+
+export function writeDataToFile(data: any, fileName: string): Promise<any> {
+  const outputPath = './test/pdf/output';

```

Comment:
il faudrait, down the road, définir une configuration d’où placer les fichier plutôt que de hard coder l'emplacement.

Reply:
fixed, j'ai ajouté une config pour le path des ouput des pdf.Je ne pense pas que ce soit nécessaire de l'ajouter dans le fichier .env non plus.

J'ai écrit ce code uniquement pour déboguer le output des PDF, donc je pense que ce code ne sera plus utilisé pendant encore longtemps (j'espère que l'ÉTS ne change pas le format de ses PDF).